### DISCUSSION TITLE
[THEME] Catppuccin

### DISCUSSION THREAD

**Discussion by SadAlexa** [TriliumNext/Trilium#4164] [Show and tell]
This is my first time creating a theme for an app. I used the Catppuccin color palette for everything so I needed it. I hope you like it!
![screenshot](https://github.com/zadam/trilium/assets/91429296/12966796-eb86-4c29-ab3f-7f95fb7c156b)

https://github.com/SadAlexa/trilium-theme-catppuccin

**Comment by Nriver** [comment]
Nice theme, thank you!

**Reply by Nriver** [reply]
addeed to https://github.com/Nriver/awesome-trilium
